lib/beaker/hypervisor/docker.rb

```diff
@@ -33,10 +33,19 @@ def provision
         image = ::Docker::Image.build(dockerfile_for(host), { :rm => true })
 
         @logger.debug("Creating container from image #{image.id}")
-        container = ::Docker::Container.create({
+
+        image_hash = {
           'Image' => image.id,
           'Hostname' => host.name,
-        })
+        }
+
+        if host[:dockeropts]
```

Comment:
This is only per-host, want to do a ```host[:dockeropts] || options[:dockeropts]``` so that you can set a default set of options globally.